src/legacy/core_plugins/kibana/public/management/index.js

```diff
@@ -171,18 +168,16 @@ uiModules.get('apps/management').directive('kbnManagementLanding', function(kbnV
   };
 });
 
-FeatureCatalogueRegistryProvider.register(() => {
-  return {
-    id: 'stack-management',
-    title: i18n.translate('kbn.stackManagement.managementLabel', {
-      defaultMessage: 'Stack Management',
-    }),
-    description: i18n.translate('kbn.stackManagement.managementDescription', {
-      defaultMessage: 'Your center console for managing the Elastic Stack.',
-    }),
-    icon: 'managementApp',
-    path: '/app/kibana#/management',
-    showOnHomePage: false,
-    category: FeatureCatalogueCategory.ADMIN,
-  };
+npSetup.plugins.home.featureCatalogue.register({
```

Comment:
There is already a `management` plugin in the new platform, you can migrate the feature registration directly.

Reply:
Thanks, done!
You could see the changes in [this commit](https://github.com/elastic/kibana/pull/56755/commits/51a9fe82c27aac199d2003b5c3bcc41c8ce38170)